tests/test_views.py

```diff
@@ -28,7 +28,7 @@ def app(dbfile):
     yield app
     # clean up / reset resources here
 
-def env_fixture(name, **kwargs):
+def env_fixture(name, **kwargs):  # place this fixture BEFORE app or its dependencies
```

Comment:
When an `env_fixture` is combined with some `app`-related fixtures (like `client`), it has to be placed before any of them to actually have any effect